cmd/argocd/commands/admin/backup.go

```diff
@@ -422,3 +431,32 @@ func updateLive(bak, live *unstructured.Unstructured, stopOperation bool) *unstr
 	}
 	return newLive
 }
+
+// updateTracking will update the tracking label and annotation in the bak resources to the
+// value of the live resource.
+func updateTracking(bak, live *unstructured.Unstructured) {
```

Comment:
`bak` doesn't seem to mean anything from the scope of this function. 
Maybe a better name would be:
```golang
func updateTracking(target, source *unstructured.Unstructured) 
```

Reply:
bak/live terminology is used everywhere in the Import command. I think we should keep it consistent with the current terminology.